.env

```diff
@@ -0,0 +1,14 @@
+MYSQLDB_LOCAL_PORT=3307
```

Comment:
Remove .env file
git rm --cached .env - .env should never be in version control
add .env.template instead with actual values removed - otherwise we would need to search for all env var throughout the project to start it